<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Debit</class>
 <widget class="QDialog" name="Debit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox_data">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>дней</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>месяцев</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>лет</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>сумма вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>срок размещения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>процентная ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>налоговая ставка ЦБ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>периодичность выплат</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_kapital">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>капитализация процентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>пополнения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>снятия</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_plus">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в полгода</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в год</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_period">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>каждый день</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>каждую неделю</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в полгода</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в год</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>в конце срока</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_minus">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в полгода</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в год</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_sum">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>9999999999999.990234375000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_data">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>100000000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_stavka">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_nalog">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_plus">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>9999999999999.990234375000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_min">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>9999999999999.990234375000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_result">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчет</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_result_procent">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>9999999999999.990234375000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>сумма налога</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_result_nalog">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>9999999999999.990234375000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>450</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>сумма на вкладе к концу срока</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_result_sum">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>480</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>9999999999999.990234375000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
